### DISCUSSION TITLE
Video for official Youtube Channel

### DISCUSSION THREAD

**Discussion by Laserlicht** [ihhub/fheroes2#6991] [Ideas]
As already discussed [here](https://github.com/ihhub/fheroes2/issues/6822).

Would be good to have official videos how to install fheroes2? Or other "how to"'s?

For many people videos are easier than with a text-based description, I think.

As discussion base here an example for Android:
https://user-images.githubusercontent.com/13953785/231552592-05ff1c11-38c0-4865-84de-ce007febc839.mp4

And an outdated example for Steam Deck (from [here](https://github.com/ihhub/fheroes2/issues/5276#issuecomment-1152619225)):
https://user-images.githubusercontent.com/13953785/173128599-c4b09f18-71f4-46f9-82cc-7effd57ca4df.mp4

Example from `heroes3wog.net`:
https://www.youtube.com/watch?v=8UBvZSg_IRc

### Ideas
**Installation:**
- [ ] Windows
- [ ] Mac OS
- [ ] Linux (Flatpak?)
- [ ] Steam Deck
- [ ] Switch
- [ ] PS Vita

Is there anyone here who can make good videos?

**Comment by zenseii** [comment]
Hi, @Laserlicht. I think this could be a good idea. We should also remember that video guides often get outdated when procedures change. So we should first discuss if we have any plans for changing the installation procedure in the near future. Afaik we only had plans to supply an updater with the app at some point.
